src/isolineFeature.js

```diff
@@ -0,0 +1,891 @@
+ *    feature.  This includes `geo.lineFeature` and `geo.textFeature` style
+ *    values.
+ * @property {number|function} [style.opacity=1] Overall opacity on a scale of
+ *    0 to 1.
+ * @property {geo.geoPosition|function} [style.position=data] The position of
+ *    each data element.  This defaults to just using `x`, `y`, and `z`
+ *    properties of the data element itself.  The position is in thea feature's
+ *    gcs coordinates.
+ * @property {number|function} [style.value=data.z] The value of each data
+ *    element.  This defaults `z` properties of the data element.  If the value
+ *    of a grid point is `null` or `undefined`, that point and elements that
+ *    use that point won't be included in the results.
+ * @property {geo.geoColor|function} [style.strokeColor='black'] Color to
+ *    stroke each line.
+ * @property {number|function} [style.strokeWidth] The weight of the line
+ *    stroke in pixels.  This defaults to the line value's level + 0.5.
+ * @property {boolean|function} [style.rotateWithMap=true] Rotate label text
+ *    when the map rotates.
+ * @property {number|function} [style.rotation] Text rotation in radians.  This
+ *    defaults to the label oriented so that top of the text is toward the
+ *    higher value.  There is a utility function that can be used for common
+ *    rotation preferences.  See {@link geo.isolineFeature#rotationFunction}.
+ *    For instance, `rotation=geo.isolineFeature.rotationFunction('map')`.
+ * @property {string|function} [style.fontSize='12px'] The font size.
+ * @property {geo.geoColor|function} [style.textStrokeColor='white'] Text
+ *    stroke color.  This adds contrast between the label and the isoline.
+ * @property {geo.geoColor|function} [style.textStrokeWidth=2] Text stroke
+ *    width in pixels.
+ * @property {geo.isolineFeature.isolineSpec} [isoline] The isoline
+ *    specification for the feature.
+ */
+
+/**
+ * Isoline specification.  All of these properties can be functions, which get
+ * passed the `geo.meshFeature.meshInfo` object.
+ *
+ * @typedef {geo.meshFeature.meshSpec} geo.isolineFeature.isolineSpec
+ * @property {number} [min] Minimum isoline value.  If unspecified, taken from
+ *    the computed minimum of the `value` style.
+ * @property {number} [max] Maximum isoline value.  If unspecified, taken from
+ *    the computed maximum of the `value` style.
+ * @property {number} [count=15] Approximate number of isolines shown through
+ *    the value range.  Used if `spacing` or `values` is not specified.
+ * @property {boolean} [autofit=true] If `count` is used to determine the
+ *    isolines, and this is truthy, the isoline values will be round numbers.
+ *    If falsy, they will include the exact minimum and maximum values.
+ * @property {number} [spacing] Distance in value units between isolines.
+ *    Used if specified and `values` is not specified.
+ * @property {number[]|geo.isolineFeature.valueEntry[]} [values] An array of
```

Comment:
Should we mention that either you can use min,max and count or values? I think it is mostly clear, just wondering if we could use bit more strong wording.

Reply:
I hope between what is written and the tutorials everything is clear enough.  Since `values` takes precedence, it doesn't mentioned `spacing` or `count`, whilst those do mention `values`.